content/docs/approval-guide.md

```diff
@@ -6,6 +6,8 @@ draft: false
 
 As a manager at Sauce Labs, you will from time to time be asked by your direct reports if they can make an internal codebase an open source project. This can be a difficult question to answer, so to guide your decision-making, we have put together a handy guide for you to review.
 
+We generally encourage approving releases unless there’s a good reason not to, but if you have any questions, you can reach out to [opensource@saucelabs.com](opensource@saucelabs.com).
```

Comment:
Does this mean that managers are encouraged to approve every release of an OSS project? If so, I think this would put too much on their workload. Maybe the intention of the phrase is different?

Reply:
This should say that the manager should approve every project that the team wants to OSS. It should not state that the team should OSS every project.

Reply:
Ok, that wording sounds much better 👍

Reply:
What do you suggest to make it more clear in the suggestion above?

Reply:
@christian-bromann what you just wrote 
>This should say that the manager should approve every project that the team wants to OSS. It should not state that the team should OSS every project.